runtime/lua/vim/lsp/protocol.lua

```diff
@@ -676,6 +676,19 @@ function protocol.make_client_capabilities()
         };
         hierarchicalDocumentSymbolSupport = true;
       };
+      workspaceSymbol = {
```

Comment:
Is this correct?

Looking at the [spec](https://microsoft.github.io/language-server-protocol/specifications/specification-current/#initialize)

```
/**
	 * Workspace specific client capabilities.
	 */
	workspace?: {
          ...

          /**
	  * Workspace specific client capabilities.
	  */
	  workspace?: {
            ...
```

I'd think this would have to look like this:


```lua
workspace = {
  symbol = {
    dynamicRegistration = false;
    symbolKind = {
      valueSet = (function()
      local res = {}
      for k in pairs(vim.lsp.protocol.SymbolKind) do
        if type(k) == 'number' then table.insert(res, k) end
      end
      return res
    end)();
  };
};
```

Reply:
Good call! It doesn't seem to make a difference for me, but better to fix it.